integration_tests/lock_test.go

```diff
@@ -893,3 +896,68 @@ func (s *testLockSuite) TestPrewriteEncountersLargerTsLock() {
 	s.Nil(failpoint.Disable("tikvclient/prewritePrimary"))
 	<-ch
 }
+
+func (s *testLockSuite) TestResolveLocksForRead() {
+	ctx := context.Background()
+	var resolvedLocks, committedLocks []uint64
+	var locks []*txnlock.Lock
+
+	// commitTS < readStartTS
+	startTS, _ := s.lockKey([]byte("k1"), []byte("v1"), []byte("k11"), []byte("v11"), true, false)
```

Comment:
Is there a test for resolving a committed async-commit lock?

Reply:
Added one https://github.com/tikv/client-go/pull/380/commits/609774e80e2b5e4bd436910f69f25b939d0d6a3e#diff-9f308ecda7da7fa7581d1f60543e34753a08fedb19e9067aa74d221421fb34f9R932